topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin.md

```diff
@@ -72,24 +82,28 @@ pluginManagement {
 >
 > To update all dependencies in the dependency cache, use the `--refresh-dependencies` command line option.
 
-
 ### Plugins
 
-The IntelliJ Platform Gradle Plugin consists of [plugins](tools_intellij_platform_gradle_plugin_plugins.md) which can be applied depending on the purpose.
-By default, the [](tools_intellij_platform_gradle_plugin_plugins.md#platform) plugin (`org.jetbrains.intellij.platform`) should be applied to the main plugin project module.
+The IntelliJ Platform Gradle Plugin consists of other [Gradle Plugins](tools_intellij_platform_gradle_plugin_plugins.md) which can be applied depending on the use case you want to fulfill.
+By default,and in the majority of cases, the [](tools_intellij_platform_gradle_plugin_plugins.md#platform) plugin (`org.jetbrains.intellij.platform`) should be applied to the main plugin project module.
 
-When working with the [](#multi-module-project-structure) it is required to use [](tools_intellij_platform_gradle_plugin_plugins.md#module) plugin (`org.jetbrains.intellij.platform.module`) instead to creating tasks and configurations specific to the main module only.
+When working in a [](#multi-module-project-structure) it is required to use [](tools_intellij_platform_gradle_plugin_plugins.md#module) plugin (`org.jetbrains.intellij.platform.module`) instead of creating tasks and configurations specific to the main module only.
 
 The [](tools_intellij_platform_gradle_plugin_plugins.md#settings) plugin (`org.jetbrains.intellij.platform.settings`) allows for adding plugin development related repositories right in the <path>settings.gradle.kts</path> file if project configuration involves [](#configuration.dependencyResolutionManagement).
 
-
 ### Attaching Sources
 
 To attach IntelliJ Platform sources in the IDE, the <control>Download sources</control> setting has to be enabled in IDE versions 2023.2 and later.
 This option respects the [](tools_intellij_platform_gradle_plugin_gradle_properties.md#downloadSources) property, which is enabled by default.
 
 <tabs>
 
+<tab title="Earlier versions">
```

Comment:
I thought those should be in chronological order since it started with the 2023.3+ tab and then a tab of 2023.2, and jumping to the earlier version is a bit confusing.

Reply:
It _is_ in chronological order. Latest version on the left, oldest on the right; this is also consistent with all other such tabs in docs. We assume that plugin developer always upgrade to new IDEs ASAP.